template.yaml

```diff
@@ -393,6 +393,24 @@ Resources:
             Condition:
               ArnLike:
                 kms:EncryptionContext:aws:dynamodb:table/arn: !Sub arn:aws:dynamodb:${AWS::Region}:${AWS::AccountId}:table/*
+          - !If
+            - IsNotDevEnvironment
```

Comment:
Why not dev?

Reply:
Because Auth stub Orch for all their dev environments, so none of Auth's lambdas will access Orch's dev environment, and this limits policies to only add access iff Auth needs it

Reply:
Auth sandpit links up to orch dev :)